### PR TITLE
Content updates

### PR DESCRIPTION
**Pull request by angelahum**
#### What's this PR do?
- Updated layout and content of FAQ and Squad pages
- Update Footer Links

#### How was this tested? How should this be reviewed?
Tested locally

### CODE REVIEW

File: assets/styles/pages/squad.less [hunk 1]
```diff
@@ -99,6 +100,14 @@ section {
   display: none;
   text-align: center;
 }
+.members {
```
Comment (jonuy) [inline assets/styles/pages/squad.less]: Looking at the page from here https://aurora-dev-pr-378.herokuapp.com/squad, I feel like the image could use a little bit of whitespace around it. What would you think about trying something like this:
```less
.members {
  margin: @margin-md;
  text-align: center;
  img {
  max-width: 600px
    width: 100%;
  }
}
```

And then you also wouldn't need the `@media (max-width: @screen-xs-max)` that follows it for the `img`. Let me know what you think.

File: views/partials/footer.ejs [hunk 2]
```diff
@@ -27,14 +28,14 @@
...
-        href="https://images.ctfassets.net/awpxl2koull4/YaA04LXbkyUsyGggOS22I/f7734cca463599e1e359e16c3d7c21f2/Shine2019HustleAndHeartReport.pdf"
-        >Hustle & Heart Report</a
+      <a href="/">Join Shine</a>
+      <a href="https://premium.shinetext.com/quiz/start"
+        >Take the Self-Care Style Quiz</a
       >
       <a href="/advice">Get Advice</a>
       <a href="/referrals">Invite Friends</a> <a href="/squad">The Squad</a>
-      <a href="/jobs">Careers</a> <a href="/faq">FAQ</a>
+      <a href="https://careers.shinetext.com/">Careers</a>
```
Comment (jonuy) [inline views/partials/footer.ejs]: Any particular reason you're making this change from /jobs? The /jobs path should already be setup on production to redirect to this careers page. If it's not working on your local setup, there's probably an environment variable that you'd need to also set for yourself.

Comment (angelahum) [reply views/partials/footer.ejs]: ohh yea, I was clicking it from my local so it didn't redirect. should I change it back?

File: assets/styles/pages/squad.less (hunk 1 above)
Comment (angelahum) [reply assets/styles/pages/squad.less]: Thanks for the suggestion!

**Comment by jonuy** [comment]
Closing this PR to be resubmitted from a different branch

**Pull request closed without merging**